css/style.css

```diff
@@ -2500,9 +2500,18 @@ footer .grid--4 {
     gap: 20px;
   }
 
+  #post-results.grid-view .article--container a {
+    display: grid;
+    grid-template-columns: 50px 1fr;
+    gap: 20px;
+  }
+
   #post-results .article--container .post-meta {
```

Comment:
I don't think we have to have separate rule for that as it should inherit rules from the rest of post-meta class.

Reply:
Sure thing, I will delete those lines and review the author.php post-meta.